atomic_reactor/plugins/pre_fetch_sources.py

```diff
@@ -229,6 +245,62 @@ def _get_remote_urls_helper(self, koji_build):
 
         return remote_sources_urls, remote_json_map
 
+    def _get_kojifile_source_urls_helper(self, koji_build):
+        """Fetch kojifile source urls from specific build
+
+        :param koji_build: dict, koji build
+        :return: list, dicts with URL pointing to kojifile sources
+        """
+
+        self.log.debug('get kojifile_source_urls: %s', koji_build['build_id'])
+        images = self.session.listArchives(koji_build['build_id'], type='image')
+
+        self.log.debug('images: %s', images)
+
+        sources = []
+        pnc_map = get_pnc(self.workflow)
+        # using urljoin here causes the API path in the base_api_url to be removed
+        api_request_url = pnc_map['base_api_url'] + '/' + pnc_map['get_scm_archive_path']
+        req_session = get_retrying_requests_session()
+
+        kojifile_build_ids = {kojifile['build_id'] for image in images
+                              for kojifile in self.session.listArchives(imageID=image['id'],
+                                                                        type='maven')}
```

Comment:
shouldn't type be 'kojifile' ???

Reply:
The way it's implemented in brew, when we add kojifile components in brew it identifies the build type of artifacts resulting in type 'maven' for fetched artifacts.

Reply:
ahh ok